### PR TITLE
Fix issue reading polylines without m values.

### PR DESCRIPTION
**Pull request by rywall**
This check already exists for points and polygons but was not added to polylines.

See:

https://github.com/culturecode/rgeo-shapefile/blob/ccf83f858fb1ba6e3b22e721aacf3a8a6746a19c/lib/rgeo/shapefile/reader.rb#L461

https://github.com/culturecode/rgeo-shapefile/blob/ccf83f858fb1ba6e3b22e721aacf3a8a6746a19c/lib/rgeo/shapefile/reader.rb#L542

### CODE REVIEW

**Comment by rywall** [comment]
@keithdoggett Would it be possible for me to become a rgeo maintainer? Let me know if you'd like any more details from me.

**Review by keithdoggett — APPROVED** [review]

**Comment by keithdoggett** [comment]
@rywall looks good thanks for the PRs! I'd like to discuss it with @BuonOmo, but I think it would be good to have you as a maintainer. Especially since you've already contributed a lot to this repo. Are you interested in working on all the gems or just rgeo-shapefile?

File: test/missing_m_value_polyline_test.rb
Comment (BuonOmo) [inline test/missing_m_value_polyline_test.rb:20]: In case we ever need to be fully windows compliant let's use the recommended methods
```suggestion
      File.expand_path(File.join("missing_m_value_polyline", name_),
      __dir__
```

**Review by BuonOmo — APPROVED** [review]
LGTM! I've offered a tiny change if you may for eventual windows compatibility.

About being a maintainer, I think this could be great indeed. to add further details on @keithdoggett's question about scope, we are thinking about having per-project maintainers which would allow us to have more time or the core RGeo. Hence the two options are definitely available : either you are responsible for this one and only or we work as a team on all projects.

On my end, I'd be glad to know a bit more about you, maybe we could continue this discussion [here](https://github.com/rgeo/rgeo/issues/216) ?

**Comment by BuonOmo** [comment]
@rywall we're merging the PR, thanks again for your contribution!

If you are still in the help us with this project or the RGeo organisation, please leave us a comment there: https://github.com/rgeo/rgeo/issues/216.

**Pull request merged**
